src/GraphQL/Types/Scalars/LongGraphType.cs

```diff
@@ -18,5 +18,8 @@ public class LongGraphType : ScalarGraphType
 
         /// <inheritdoc/>
         public override object ParseValue(object value) => ValueConverter.ConvertTo(value, typeof(long));
+
+        /// <inheritdoc/>
+        public override bool CanParseLiteral(IValue value) => value is IntValue || value is LongValue;
```

Comment:
I would prefer to keep `switch value` style everywhere even in the simplest cases.

Reply:
OK, nevermind.